plugins/woocommerce/client/admin/client/payments/test/payment-recommendations.test.tsx

```diff
@@ -175,6 +183,25 @@ describe( 'Payment recommendations', () => {
 		expect( container.firstChild ).toBeNull();
 	} );
 
+	it( 'should trigger event settings_payment_recommendations_visit_marketplace_click when clicking the Official WooCommerce Marketplace link', () => {
```

Comment:
The other case to consider is that the href changes properly on click, do you want to capture that in tests as well?

Reply:
That's worth testing, yes. Good suggestion.

Reply:
Tests added in 0260786